src/Tools/perf/ExternalProcessBenchmark.cs

```diff
@@ -0,0 +1,42 @@
+﻿
+using System;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Parameters;
+using BenchmarkDotNet.Running;
+
+namespace perf
+{
+    internal sealed class ExternalProcessBenchmark : Benchmark
+    {
+        public string WorkingDirectory { get; }
+
+        public string Arguments { get; }
+
+        public Func<string, int> BuildFunc { get; }
+
+        public ExternalProcessBenchmark(
+            string workingDir,
+            string arguments,
+            Func<string, int> buildFunc,
+            Job job,
+            ParameterInstances parameterInstances)
+        : base(GetTarget(), job, parameterInstances)
+        {
+            WorkingDirectory = workingDir;
+            Arguments = arguments;
+            BuildFunc = buildFunc;
+        }
+
+        private static Target GetTarget()
```

Comment:
Might want to give a comment "why" here. I can imagine why, but better to not leave me guessing.